cpp/CMakeLists.txt

```diff
@@ -188,6 +191,16 @@ GENERATE_FIND_MODULE(NAME UCX
                      HEADER_NAME ucp/api/ucp.h
                      LIBRARY_NAME ucp)
 
+if(NOT BUILD_GTEST)
+	GENERATE_FIND_MODULE(NAME GTEST
```

Comment:
We actually don’t need to use the template here, you can do `find_package(GTest)` natively in cmake